lms/djangoapps/grades/tests/test_grades.py

```diff
@@ -222,51 +228,134 @@ def create_location(self, block_type, block_id):
         )
 
     def test_score_chapter(self):
-        earned, possible = self.progress_summary.score_for_module(self.loc_a)
+        earned, possible = self.course_grade.score_for_module(self.loc_a)
```

Comment:
I agree with the spirit of what you're suggesting, but there's a good bit of clarifying documentation in the class docstring to explain the block hierarchy that I think would become more confusing with longer-named blocks.